src/Native/Common/CPPCodegen/common.h

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
> src/Native/Common/CPPCodegen/common.h 
> src/Native/Common/CPPCodegen/platform.h

The current copies of these files live under native\bootstrap. We should just pick them from there, at least for now.